src/routes/userRouter.js

```diff
@@ -1,18 +1,22 @@
 const express = require("express");
 
-const controller = require("../controller/userController");
+const userController = require("../controller/userController");
+const { isAuthenticated } = require("../middlewares/authenticationMiddleware");
 const route = express.Router();
 
-// Add New User
-route.post("/", controller.adddata);
+// Always Keep Generic Routes on top
+// Login authRoute
+route.post("/login", userController.login);
+// Logout authRoute
+route.get("/logout", userController.logout);
 
+// Add New User
+route.post("/", isAuthenticated, userController.adddata);
```

Comment:
how can we provide token while creating new user?

Reply:
we can't add authentication middleware here because after creating user and login we will get token